packages/cli-platform-android/native_modules.gradle

```diff
@@ -366,29 +366,20 @@ class ReactNativeModules {
   }
 
   /**
-   * Runs a specified command using Runtime exec() in a specified directory.
+   * Runs a specified command using providers.exec in a specified directory.
    * Throws when the command result is empty.
    */
   String getCommandOutput(String[] command, File directory) {
     try {
-      def output = ""
-      def cmdProcess = Runtime.getRuntime().exec(command, null, directory)
-      def bufferedReader = new BufferedReader(new InputStreamReader(cmdProcess.getInputStream()))
-      def buff = ""
-      def readBuffer = new StringBuffer()
-      while ((buff = bufferedReader.readLine()) != null) {
-        readBuffer.append(buff)
+      def execOutput = providers.exec {
```

Comment:
CI fails with this error on this line: 

<img width="1090" alt="image" src="https://github.com/react-native-community/cli/assets/5106466/9be4e8d3-6138-4717-a6f6-b25a5132cfaa">

Reply:
ah - created the PR from the wrong branch, should be fixed with https://github.com/react-native-community/cli/pull/2201/commits/33fa4a1e42b0c2ad59a769986199e114d5767afa